CONTRIBUTING.md

```diff
@@ -23,7 +23,6 @@
 >
 > _Director, ADL Initiative_
 >
-> _OSD, Training Readiness & Strategy (TRS)_
```

Comment:
That is not the organization we fall under. I think just having it as Director, ADL Initiative should be be here. I will also reach out to our new director about adding a message, but for now just wanted to update this for accuracy.